docs/api-methods/30_sign-message.md

```diff
@@ -75,8 +75,8 @@ The method returns a `SignedMessage` object containing the following properties:
 ```javascript
 interface SignedMessage {
     signer: string;               // Userfriendly address
-    signerPublicKey: Uint8Array;  // The public key of the signer
-    signature: Uint8Array;        // Signature for the message
+    signerPublicKey: number[];  // The public key of the signer
+    signature: number[];        // Signature for the message
```

Comment:
Should state which values are to be expected here, i.e. integers in the range 0..255.